README.md

```diff
@@ -1 +1,38 @@
 # hw01
+## 10/04/2020
+***
+
+Hi everyone, my name is Emily Robb and I'm a fourth year in the College. I'm double-majoring in Philosophy and Political Science and am incredibly excited to take this course and learn practical skills that I haven't really had previous exposure to. Looking forward to learning with all of you!
+
+![Picture of me](IMG_5373.jpeg)
+
+---
+
+##### Process Description
+I started this homework by reviewing the overview and instructions, and ultimately watching the accompanying video.
+
+[Panopto Instructions](https://uchicago.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=7bd284fa-ceac-400b-a8db-ab9500f8d943&start=199.610526)
```

Comment:
To make the markdown more seamless, you can actually hyperlink the video instruction link by simply making the 'Panopto instructions' segment a part of your sentence!